src/pages/api/hello.js

```diff
@@ -1,5 +1,5 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
 
 export default function handler(req, res) {
-  res.status(200).json({ name: 'John Doe' })
+  res.status(200).json({ name: 'John Doe' });
```

Comment:
do we need this api route as part of this task ?

Reply:
It was generated with nextJs but I don't think we need this.